.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+sudo: false
+node_js:
+  - 5
+env:
+  - NODE_ENV="test"
+#before_script:
+  #- npm run db:setup:user
+  #- npm run db:reset
+script:
+  - npm test
+  #- npm run enforce
```

Comment:
minor but probably remove the lines entirely until you're ready to add them back in